swervelib/ctre/KrakenX60DriveControllerFactoryBuilder.java

```diff
@@ -0,0 +1,120 @@
+package frc.team670.mustanglib.swervelib.ctre;
+
+import com.ctre.phoenix.motorcontrol.*;
+import com.ctre.phoenix.motorcontrol.can.TalonFXConfiguration;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+import frc.team670.mustanglib.swervelib.DriveController;
+import frc.team670.mustanglib.swervelib.DriveControllerFactory;
+import frc.team670.mustanglib.swervelib.ModuleConfiguration;
+
+public final class KrakenX60DriveControllerFactoryBuilder {
+    // REPLACE CONSTANTS
```

Comment:
what does this mean? are these constants valid or are they not?

Reply:
It means they are invalid